osu.Game/Overlays/Practice/PracticeOverlayComponents/PracticeRangeSliderComponent.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Events;
+using osu.Framework.Localisation;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.Practice.PracticeOverlayComponents
+{
+    public class PracticeRangeSliderComponent : CompositeDrawable
```

Comment:
thisclass seems like mostluy copy paste of another class. please don't do this.

Reply:
yep i was planning to merge them together ( had trouble doing it initially)  ill add it to the todo list :)